splot/_viz_value_by_alpha.py

```diff
@@ -0,0 +1,36 @@
+import geopandas as gpd
+from shapely.geometry import Polygon
+import matplotlib
+
+"""
+Creating Value by Alpha maps
+
+"""
+
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+
+def value_by_alpha(x, alphay, gdf):
```

Comment:
this should get dropped

Reply:
sorry, I should have added `value_by_alpha` functionality is dealt with separately in the next pr #28